### ISSUE TITLE
Can only save 1 paste?

### ISSUE THREAD

**Issue by tmpm697** [yory8/clipman#64]
I notice that whatever how many copy I made, I can only see the last copy in clipmenu with bemenu.

Is this wayland limit or if not how to have more items?

Info:
```
os: archlinux up-to-date
clipman ver: 1.5.2
```

**Comment by yory8** [comment]
I cannot reproduce. 
This is not clipmenu but clipman

**Comment by tmpm697** [comment]
I have this setup:
```
vi sway/config
exec wl-paste -t text --watch clipman store

shell> clipman pick --print0 --tool=CUSTOM --max-items=30 --tool-args="bemenu -l 6"
```
Now I only see last copy in clipboard and even -l 6 should show 6 items but it shows only one last copy.
(I used left mouse to select number of text on screen to put them in to clipboard)

**Comment by yory8** [comment]
`--tool=bemenu --tool-args="-l 6"` should do the trick

**Comment by tmpm697** [comment]
@yory8 : but bemenu seems to always displays at bottom of screen if use with `clipmand`, how to have it display at top (as its default)?

**Comment by tmpm697** [comment]
> @yory8 : but bemenu seems to always displays at bottom of screen if use with `clipmand`, how to have it display at top (as its default)?

seem like it's fixed in #54
